components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -2177,6 +2178,7 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
 
         const user = this.checkAndBlockUser("subscribeToStripe");
 
+        let team: Team | undefined;
         try {
             if (attrId.kind === "team") {
                 const team = await this.guardTeamOperation(attrId.teamId, "update");
```

Comment:
I this that when this is declared as a `const` it will shadow the `let team` above and won't actually get assigned

Reply:
Oops good catch!

Reply:
Fixed - thanks 🙏🏽